.circleci/config.yml

```diff
@@ -25,11 +36,79 @@ jobs:
       - store_artifacts:
           path: build/reports/tests
 
+  build:
```

Comment:
I forget why it is we's not using the `hmpps/validate` job here 🤔  I know there was a reason..

Reply:
Perhaps we should use it but it just does test as far as i can see with 

`-Dorg.gradle.jvmargs="--illegal-access=permit" -Dkotlin.daemon.jvm.options="--illegal-access=permit"`

I don't think we need that.

Reply:
Shall I remove it ?

Reply:
Yeah if you don't mind, would be good to use as much stuff as the orb gives us as possible

Reply:
Is ours doing anything else other than test?

Reply:
Our build phase does test and build